requirements-docs.txt

```
@@ -1,6 +1,6 @@
 ablog>=0.11
 matplotlib
-myst-nb
+myst-nb<=1.0.0
```

```suggestion
myst-nb!=1.1
```

There is already a PR open with the fix, so I think we can do this so we don't have to undo the pin, simply indicate this specific version is faulty. What do you think? Either way should work though, just trying to ponder which is more probable to end up with less future work